queue_job/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+
+{'name': 'Job Queue',
+ 'version': '10.0.1.0.0',
+ 'author': 'Camptocamp,Acsone,Odoo Community Association (OCA)',
+ 'website': 'http://odoo-connector.com',
+ 'license': 'AGPL-3',
+ 'category': 'Generic Modules',
+ 'depends': ['mail'
+             ],
+ 'external_dependencies': {'python': ['requests'
+                                      ],
+                           },
+ 'data': ['security/security.xml',
+          'security/ir.model.access.csv',
+          'views/queue_job_views.xml',
+          'data/queue_data.xml',
+          ],
+ 'installable': True,
+ 'application': True,
```

Comment:
Even if it does have a top level menu, I'm not sure it's enough to call it an application.

Personally, I'd be ok if the Queue menu was somewhere inside Technical instead of being top-level.

If I need to keep an eye on the jobs queue all the time so that a top-level menu makes sense, then I'd argue something is wrong in my application...